relay-server/src/metrics_extraction/spans/mod.rs

```diff
@@ -261,6 +269,40 @@ pub(crate) fn extract_span_metrics(
     Ok(())
 }
 
+/// Returns the sanitized span description.
+///
+/// If a scrub description is provided, that's returned. If not, a new
+/// description is built for `http*` modules with the following format:
+/// `{action} {domain}/<unparameterized>`.
```

Comment:
I guess I could write a nice doctest here, but I didn't want to put that much effort.